src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.android.js

```diff
@@ -54,12 +55,18 @@
           onpollchanged={(v) => {
             onsinglepollchanged(v, index);
           }}
+          isAnonym={isAnonym}
         />
       ))}
 
       {polls.length < MAX_POLLING_ALLOWED && (
         <TouchableOpacity onPress={() => onaddpoll()} style={S.addpollitemcontainer}>
-          <MemoIcPlus width={16} height={16} style={S.addpollitemplusicon} />
+          <MemoIcPlus
+            width={16}
+            height={16}
+            style={S.addpollitemplusicon}
+            fill={isAnonym ? COLORS.anon_secondary : COLORS.signed_secondary}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnonym` prop is being used to conditionally render styles. This is a good practice for dynamic styling based on certain conditions. However, ensure that the `COLORS.anon_secondary` and `COLORS.signed_secondary` are defined and have valid color codes.
```diff
-         />
+         isAnonym={isAnonym}
+        />
+  
+        {polls.length < MAX_POLLING_ALLOWED && (
+          <TouchableOpacity onPress={() => onaddpoll()} style={S.addpollitemcontainer}>
+           <MemoIcPlus
+             width={16}
+             height={16}
+             style={S.addpollitemplusicon}
+             fill={isAnonym ? COLORS.anon_secondary : COLORS.signed_secondary}
+           />
+         </TouchableOpacity>
+       )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->